### PR TITLE
[TR] PIM-4349: Allow to look for 0 notification

### PR DESCRIPTION
**Pull request by grena**
| Q                 | A
| ----------------- | ---
| Added Specs       | N
| Added Behats      | N
| Travis CI is ok   | Y
| Changelog updated | N

### CODE REVIEW

**Comment by solivier** [comment]
GTM

File: features/Context/AssertionContext.php [hunk 1]
```diff
@@ -596,6 +596,10 @@ public function theScopableFieldShouldHaveTheFollowingColors($field, TableNode $
     public function iShouldHaveNewNotification($count)
     {
         $actualCount = $this->getCurrentPage()->find('css', '#header-notification-widget .indicator .badge')->getText();
```
Comment (jjanvier) [inline features/Context/AssertionContext.php]: cast to int instead

**Comment by jjanvier** [comment]
not gtm

**Comment by jjanvier** [comment]
gtm sale

**Pull request merged**